admin/build-doc

```diff
@@ -22,7 +22,11 @@ if command -v dpkg >/dev/null; then
 elif command -v yum >/dev/null; then
     for package in python-devel python-pip python-virtualenv doxygen ditaa ant libxml2-devel libxslt-devel Cython graphviz; do
 	if ! rpm -q $package >/dev/null ; then
-		missing="${missing:+$missing }$package"
+		command -v `echo $package | tr '[:upper:]' '[:lower:]' | awk -F "-" '{ if ($2 == "") print $1; else print $2}'`   > /dev/null;
```

Comment:
If I am understanding this correctly, you are adding this multi-pipe with `tr` and `awk` to verify if the executable exists, inferring the name of the executable from the package, right?

I think this is still a brittle way to verify this. There is more than one package that doesn't have an executable, or that the executable that your pipe line is trying to parse is incorrect (e.g. python-devel would parse as 'devel'). 

The problem here is not that the executable may be there and we should assume then that the package is installed, the problem here is that you need to have the right packages for FC25, which this script doesn't account for.

If it had the right packages for your system, then this parsing wouldn't be necessary.